cc.nift.nsm.json

```diff
@@ -0,0 +1,25 @@
+{
+    "app-id": "cc.nift.nsm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "nsm",
+    "modules": [
+        {
+            "name": "nsm",
+            "buildsystem": "simple",
+            "build-commands": [
+				"make",
+				"make install"
+            ],
+            "sources": [
+                {
+					"type": "git",
+					"url": "https://github.com/nifty-site-manager/nsm-flatpak.git",
```

Comment:
The repo split is just *wrong*. You don't copy paste your sources for every package format its entirely ridiculous.

Reply:
The install/uninstall part of the makefile is different, I have windows/osx/linux install functions for the Makefile for nifty-site-manager/nsm repo. I'm keen for nsm to be available on FlatHub which some people have expressed interest in, but not keen enough to have the main install function in the Makefile be specifically for FlatHub. It's very easy to keep a separate repo for nsm-flathub/pak, I did similarly without problem for snapcraft and chocolatey.